workflows/templates/register-indexes-template.yaml

```diff
@@ -0,0 +1,58 @@
+---
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: register-indexes
+  annotations:
+    thoth-station.ninja/template-version: 0.1.0
+  labels:
+    app: thoth
+    component: register-indexes
+spec:
+  templates:
+    - name: register-indexes
```

Comment:
why is workflows/templates/solve-core-packages.yaml the same?

Reply:
Corrected